src/python/dxpy/bindings/dxapplet.py

```diff
@@ -192,6 +195,8 @@ def run(self, executable_input, project=None, folder=None, name=None, tags=None,
         :type ignore_reuse_stages: list
         :param extra_args: If provided, a hash of options that will be merged into the underlying JSON given for the API call
         :type extra_args: dict
+        :param detach: If provided, job will not start as subjob if run inside of a different job.
```

Comment:
Please change detach location here in docstrings, too.

Reply:
fixed